frontend/src/constants.ts

```diff
@@ -1,6 +1,7 @@
 import { Network } from "@aptos-labs/ts-sdk";
 
-export const defaultNetwork = Network.MAINNET;
+export type Env = "prod" | "staging";
```

Comment:
We don't have envs for this, can we remove this?

Reply:
Sure